protokube/cmd/protokube/main.go

```diff
@@ -188,7 +199,7 @@ func run() error {
 	}
 
 	rootfs := "/"
-	if containerized {
+	if containerized && cloud != "vsphere" {
```

Comment:
/rootfs is being used for docker volume mounting. I have removed my changes that were altering the containerized behavior.